src/components/ItemDetail.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import arrowLeft from '../image/arrow-left.svg';
+import '../CSS/Icon.css';
+
+const ItemDetail = ({ list }) => {
+  let { itemName } = useParams();
+  let item = list.find(item => item.name_normalized === itemName);
+
+  let lastPurchase = item ? new Date(item.last_purchased).toDateString() : '';
+
+  return item ? (
+    <main>
```

Comment:
I would consider having this `main` tag be at the top level of the content of the entire App, rather than only on the Item Details page. So that the HTML would look like:
`<body>`
     `<Header/>`
     `<main>`
            `whichever content is required for that particular page like <ItemDetail/>`
     `</main>`
     `<footer/>`
`</body>`
Not a blocker for this ticket, but something to think about for the last week.